src/chainparams.cpp

```diff
@@ -64,6 +64,7 @@ class CMainParams : public CChainParams {
         consensus.BlockV9TallyHeight = 1144120;
         consensus.BlockV10Height = 1420000;
         consensus.BlockV11Height = 2053000;
+        consensus.powLimit = uint256S("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
```

Comment:
You should retain the commenting on why these values are what they are (and what they are in compact notation...)